urls/funcmap.go

```diff
@@ -86,3 +90,38 @@ func queryClear(u *url.URL) (*url.URL, error) {
 
 	return &newU, nil
 }
+
+func renderMsg(oldVal string) template.HTML {
+	oldVal = html.EscapeString(oldVal)
+
+	oldVal = strings.ReplaceAll(oldVal, "\r\n", "\n")
```

Comment:
Why is this necessary?

Reply:
I'm just not sure that all newlines are windows/internet newlines (where `\r\n` is used). Better safe than sorry?